templates/legal/ubuntu-advantage-service-description/index.md

```diff
@@ -4,386 +4,390 @@ context:
+6. Support language. Canonical will provide the support in English, unless specified otherwise
+7. Remote sessions. At the discretion of a Canonical engineer, a remote access service might be offered to access a supported system. In such a case, Canonical will determine which remote access service to use. Canonical engineers do not perform any remote actions on a supported system
+8. Ask for a Peer Review. As a normal business practice, Canonical performs peer reviews on a percentage of all cases. Customers can specifically request a peer review on a case within the case comments or by calling the phone number listed in the support portal. An impartial engineer will be assigned to review the case and provide feedback
+9. Management escalation. The customer may escalate support issues following the escalation process:
+    14. Non-urgent needs. Request a management escalation within the case itself. A manager will be contacted to review the case and post a response within 1 business day
+    15. Urgent needs can be escalated to Canonical’s Support & Technical Services Manager by emailing [support-manager@canonical.com](mailto:support-manager@canonical.com). If you require further escalation, email Canonical’s Support & Technical Services Director at [operations-director@canonical.com](mailto:operations-director@canonical.com)
 
 <div class="p-top"><a href="#" class="p-top__link">Back to top</a></div>
 
-<h2 id="uasd-desktop-support">Desktop support</h2>
+<h2 id="uasd-definitions">Definitions</h2>
 
-1. Desktop support services are subject to the exceptions listed below.
-2. Scope. Desktop support covers all packages in the base Ubuntu desktop image and packages necessary for basic network authentication and connectivity using sssd, winbind, network-manager, and network-manager plugins in the Ubuntu Main repository. It also covers Ubuntu distribution for Windows Subsystem for Linux (WSL).
-3. Out of scope.
-4. Dual-booting (cohabitating with other operating systems).
-5. Peripherals which are not certified to work with Ubuntu.
-6. Community flavours of Ubuntu.
-7. Support for architectures other than x86_64.
+**Applications:** the supported Applications as set out at [https://ubuntu.com/support](https://ubuntu.com/support) [which are built by Canonical or as otherwise agreed with Canonical. Or Managed Applications as described in the Add-ons section, under Managed Services, and at https://ubuntu.com/managed](https://ubuntu.com/managed).[ ](https://ubuntu.com/managed) The list of Applications are subject to change without notice
```

Comment:
the link is not this entire thing, remove the brackets. Also `[ ](https://ubuntu.com/managed)` doesn't look like correct markdown

Reply:
Good spot thanks @carkod - fixed